static/src/stylesheets/head.content.scss

```diff
@@ -36,6 +37,9 @@
 @import 'module/external/_from-content-api';
 @import 'module/content/_badging';
 
+// Needed for more seamless loading of Hosted By page (Renault release)
```

Comment:
Just let's make sure that in the future we will create own head.commercial.scss file which will be loaded only on this hosted pages and not globally. These files needs to be kept minimal.